app/_src/mesh/guides/create-a-mesh-using-terraform-and-konnect.md

```diff
@@ -0,0 +1,602 @@
+---
+title: Create a Mesh using Terraform and Konnect
```

Comment:
"Create a Mesh" sounds a bit odd, should this be "Manage Mesh using Terraform and Konnect" as you have in the sidebar?

Reply:
I wanted to do another guide on doing `terraform import` that would show how to import an existing setup into terraform. How would you call that one if we keep "manage"?

Reply:
My problem is probably with the word "a Mesh". Do you mean it here as a product, i.e., "Service Mesh" or "Kong Mesh", or as a Mesh resource? 

Also, shouldn't the title in the sidebar be the same as the guide's title?

Reply:
Agreed on "Deploy Kong Mesh using Terraform and Konnect"